frontend/src/components/static_pages/about.css

```diff
@@ -0,0 +1,43 @@
+.images {
+  width: 400px;
+  margin: 5%;
+}
+
+.lowermid {
+  background-color: #eeeaea;
+  width: 100%;
+}
+
+.sub-heading {
+  font-size: 24pt;
+}
+
+/* tablet */
+
+@media all and (min-width: 426px) and (max-width: 768px) {
```

Comment:
seeing a bit of a problem around 769px  - 980px
![image](https://user-images.githubusercontent.com/1637481/62503205-c6e0d680-b7b8-11e9-8cfc-acd33c350afd.png)

there is a few ways to solve this. I believe you can manipulate `md` in the html or can do it via css -> i will leave it up to you 😎